jbmc/unit/java_bytecode/java_static_initializers/java_static_initializers.cpp

```diff
@@ -24,3 +24,22 @@ SCENARIO("is_clinit_function", "[core][java_static_initializers]")
     REQUIRE_FALSE(is_clinit_function(input));
   }
 }
+
+SCENARIO(
+  "is_user_specified_clinit_function",
+  "[core][java_static_initializers]")
+{
+  GIVEN("A function id that represents a user-specified clinit")
+  THEN("is_user_specified_clinit_function should return true.")
```

Comment:
ℹ️ This is not what GIVEN and THEN are for. For such simle functions, I'd use TEST_CASE.